### PR TITLE
Ports gloves of the north star from Hippie

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: GuyonBroadway
add: Adds gloves of the north star to the traitor uplink. Wearing these gloves lets you punch people five times faster than normal.
rscadd: Sprites for the gloves courtesy of Notamaniac
/:cl:

Discord told me to do it.

-It only applies to harm intent attacks.
-It does not increase attack speed with held objects.
-It does not increase hulk attack speed due to how hulks work (and its probably for the best)
-It does work with krav maga in theory but seeing as there isn't currently an in game way (on tg at least) to get krav maga without also wearing the krav maga gloves it doesn't work in practise.
-It DOES work with sleeping carp but at 8tc you will need some luck with discounts in order to be able to use both, also see here for why this might not be as overpowered as you might think. https://youtu.be/Kp09QB5-Z3E

~~Yes the sprites aren't hand-wraps like wot people tell me kenny boy wore in that thar anny may but I'm not a spriter and have less than 0 artistic talent. If you want to make some nice sprites to go with these then be my guest, I will however steal them for hippie.~~

Disregard, Notamaniac made some nice sprites that serve as a decent visual indicator that someone is wearing gloves of beating the shit our of you... as if a guy screaming ATATATATATATATATATATATAT and punching faster than motorhead's drummer wasn't a big enough indicator.

### CODE REVIEW

**Comment by Spaceinaba** [comment]
\>it doesnt make people explode after you punch them enough times
DO NOT MERGE

**Comment by epochayur** [comment]
you are already closed

File: code/modules/uplink/uplink_items.dm
Comment (GuyonBroadway) [inline code/modules/uplink/uplink_items.dm]: Whoops forgot the extra line

File: code/modules/clothing/gloves/miscellaneous.dm
Comment (Iamgoofball) [reply code/modules/clothing/gloves/miscellaneous.dm]: actually hm
we should count the number of clicks and generate it that way

**Comment by GuppyLaxx** [comment]
OH YES PLEASE

**Comment by rustledjimm** [comment]
>/tg/ porting anime stuff from Hippie

Wait a minute....

File: code/modules/clothing/gloves/miscellaneous.dm
Comment (Kawaii-Big-Boss) [reply code/modules/clothing/gloves/miscellaneous.dm]: time to find out how many people use macros when they end up screeching "ATATATATAT" 900,000 times in a second

Comment (steamp0rt) [reply code/modules/clothing/gloves/miscellaneous.dm]: Actually, `M.say("AT", ignore_spam = TRUE)`

**Comment by drderp3635** [comment]
Make a way that you can punch without dealing damage, but after hitting a certain threshold maybe 20-30 punches it will cause them to explode without gibbing

**Comment by Wilchenx** [comment]
I would say definitely block sleeping carp usage with this.

**Comment by GuyonBroadway** [comment]
@Wilchenx It's already "soft blocked" by way of Carp costing 17 and this costing 8 needing a pretty decent discount on either in order to use both (or manage to convince a fellow traitor to hand over his tc so you can have all the fun but good fucking luck with that) so if the starts DO align being able to run a gimmick once in a while shouldn't be too bad.

Not to mention that using both isn't as strong as you might think, firstly it only works with harm intent so carp combo's aren't executed any faster. Secondly spending pretty much all your TC on a gimmick that is ABOUT as strong as a dual-e but with no melee block chance is usually a good way to get yourself slipcuffed or just have one plucky sec officer lightly tap you with the business end of a baton and laugh at you. Not to mention borgs will think it just tickles and carry on their day, probably flashing you and dragging you to the sec office.

Don't get me wrong you can beat people seven ways from sunday if you get a few good hits in and being able to act out your animeme fantasies on the station is a pretty big powertrip but over on hippie where you can get carp and gloves EVERY ROUND thanks to tc price rebalancing we very quickly found out that spending all your tc on a gimmick tends to go sour more times than it works out for you.

**Comment by Shadowflame909** [comment]
If I get this, sleeping carp, and a stand. Will I be the anime shitpost God?

**Comment by Notamaniac** [comment]
Here's some shit sprites until someone comes along with something better

meh
bad
West
badw
North
badn
South
bads
East
bade

File: code/modules/clothing/gloves/miscellaneous.dm
Comment (Cyberboss) [reply code/modules/clothing/gloves/miscellaneous.dm]: ^ I'm serious about this btw, we have chat squashing

**Comment by epochayur** [comment]
those look pretty damn nice, better than what i was working on

File: code/modules/clothing/gloves/miscellaneous.dm
Comment (GuyonBroadway) [reply code/modules/clothing/gloves/miscellaneous.dm]: Gonna add the custom four letter warcry at some point soonish...

Comment (GuyonBroadway) [reply code/modules/clothing/gloves/miscellaneous.dm]: >totally not just lifting this right from holoparasite code but whatever.

**Comment by GuyonBroadway** [comment]
As part of an unintended side effect, someone wearing GOTN will shout their battlecry any time they click on something with harm intent.

FEATURE!!

**Comment by lordsaladin** [comment]
What's next saitama mid round antag? so we can one punch everything?

**Comment by GuyonBroadway** [comment]
@lordsaladin Heres something else I coded for hippie
https://youtu.be/P2NFpBdPlUc

**Comment by Shadowflame909** [comment]
This hasn't been merged yet because the maintainers know. Once this gets merged it is time for the hippie nooses PR.

File: code/modules/clothing/gloves/miscellaneous.dm
Comment (SpaceManiac) [inline code/modules/clothing/gloves/miscellaneous.dm]: The warcry sounds when clicking anywhere on-screen with harm intent, including machinery and areas out of vision, is this intended?

File: code/modules/clothing/gloves/miscellaneous.dm
Comment (SpaceManiac) [inline code/modules/clothing/gloves/miscellaneous.dm]: "Gloves of the North Star" or change the uplink item name to match this.

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

File: code/modules/clothing/gloves/miscellaneous.dm
Comment (GuyonBroadway) [reply code/modules/clothing/gloves/miscellaneous.dm]: It only does that when the touch action is applies which is when you click on something with an empty hand while wearing gloves.

Also yes very much intended as its funny, just the idea of someone yelling _ATATATATATATATATATATATAT!_ while typing on the HoP computer made me giggle in a way an adult man shouldn't.

File: code/modules/clothing/gloves/miscellaneous.dm
Comment (SpaceManiac) [inline code/modules/clothing/gloves/miscellaneous.dm]: North Star with an N, otherwise PR looks good.

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

**Review by SpaceManiac — APPROVED** [review]
Had to resort to the webeditor to fix the grammar myself, otherwise I would have done it from command-line days ago.

**Pull request merged**
